### PR TITLE
update file format aobj

### PR DESCRIPTION
**Pull request by Olivia266**

### CODE REVIEW

**Review by albertmink — APPROVED** [review]
LGTM, but let's wait for another reviewer.
Did you consider to provide an example as well, see #387 ?

File: file-formats/aobj/aobj-v1.json [hunk 1]
```diff
@@ -78,12 +78,19 @@
                     "description": "Name of executable class or program which reloads the data to the tables (may be empty)",
                     "type": "string",
                     "maxLength": 30
+                },
+                "storageClass": {
+                    "title": "Storage Class",
+                    "description": "Name of class which stores and retrieves archived data. The class needs to implement the interface IF_ARCH_STORAGE_MANAGER.",
```
Comment (albertmink) [inline file-formats/aobj/aobj-v1.json:84]: @schneidermic0 do you have an option on the reference to the `INTF`? Typically, the description does not refer to technical details.

Comment (schneidermic0) [reply file-formats/aobj/aobj-v1.json:84]: I think this information is fine for me.

**Review by schneidermic0 — APPROVED** [review]
Thanks. Looks good to me.

**Review by wurzka — APPROVED** [review]

**Pull request merged**